Plots/XY/NCL_xy_2.py

```diff
@@ -0,0 +1,64 @@
+"""
+NCL_xy_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing an XY plot with multiple curves
+   - Changing the line color for multiple curves in an XY plot
+   - Changing the line thickness for multiple curves in an XY plot
+   - Drawing XY plot curves with both lines and markers
+   - Changing the default markers in an XY plot
+   - Making all curves in an XY plot solid.
+See https://www.ncl.ucar.edu/Applications/xy.shtml for further information.
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tic
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset("../../data/netcdf_files/uv300.nc")
+u = ds.U[:]
+
+###############################################################################
+# Generate plot data from netCDF data
+# xx = u.values[0,:,82]
+xx = u[0,:,:].sel(lon=[82], method='nearest')
+# yy = u.values[0,:,-69]
+yy = u[0,:,:].sel(lon=[-69], method='nearest')
+lat = u['lat'].values
+
+###############################################################################
+# Multiple line plot
+fig = plt.figure()
+ax = plt.axes()
```

Comment:
after the `.plot()` calls, this could be

```
fig = plt.gcf()  # get current figure
ax = plt.gca()  # get current axes
```

though `fig` is unnecessary since it isn't used anywhere

Reply:
Agree and pushed the addressing changes.